src/tests/nativeaot/SmokeTests/HelloWasm/HelloWasm.csproj

```diff
@@ -22,6 +22,6 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <EmccExtraArgs>--js-library $(MSBuildProjectDirectory)\dotnet_support.js --pre-js $(MSBuildProjectDirectory)\Microsoft.JSInterop.js --post-js $(MSBuildProjectDirectory)\HelloWasm.js</EmccExtraArgs>
+    <EmccExtraArgs>--js-library &quot;$(MSBuildProjectDirectory)\dotnet_support.js&quot; --pre-js &quot;$(MSBuildProjectDirectory)\Microsoft.JSInterop.js&quot; --post-js &quot;$(MSBuildProjectDirectory)\HelloWasm.js&quot;</EmccExtraArgs>
```

Comment:
Can we avoid this breaking change by passing `EmccExtraArgs` to `emcc` directly: `"$(EmccPath) @$(NativeIntermediateOutputPath)link.rsp $(EmccExtraArgs)"`?